modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-install-kernelrt-rhel-edge_{context}"]
+= Installing the {op-system-rt-kernel} in a {op-system-ostree-first} image
+
+You can include the real-time kernel in a {op-system-ostree} image deployment using image builder.
+
+.Prerequisites
+* You have a Red Hat subscription enabled on the host that includes {op-system-rt-kernel}.
+* You are using the x86_64 architecture.
+* You configured `osbuild` to use the `kernel-rt` repository.
+
+[NOTE]
+====
+The following example blueprint sections include references gathered from the previous steps required to configure low latency for a {microshift-short} cluster.
+====
+
+[IMPORTANT]
+====
+A subscription that includes the {op-system-rtk} must be enabled on the host used to build the commit.
+====
```

Comment:
Is this different than the first prereq? If so, maybe combine them and remove the stacked admonition?

Reply:
It's the same, but there's a sep task from Support and QE to make this note very very prominent. Removed the stacked admonition, tho.